qiskit/pulse/schedule.py

```diff
@@ -324,34 +331,51 @@ def _construct_filter(self, *filter_funcs: List[Callable],
             time_ranges: For example, [(0, 5), (6, 10)]
             intervals: For example, [Interval(0, 5), Interval(6, 10)]
         """
-        def only_channels(channels: Set[Channel]) -> Callable:
+        def only_channels(channels: Channel) -> Callable:
```

Comment:
I believe this is still accepting an iterable? Should this be a type of `Union[Set[Channel], Channel]`.